.prettierignore

```diff
@@ -21,3 +21,4 @@ test-timings.json
 test/**/out/**
 bench/nested-deps/pages/**/*
 bench/nested-deps/components/**/*
+examples/with-temporal/temporal/lib
```

Comment:
Should this be added in the example code instead?

Reply:
I tried that, but didn't work! Looks like prettier [only looks at the root `.prettierignore`](https://github.com/prettier/prettier/issues/4081)

Reply:
I will add it there in addition, so it's in the `create-next-app -e with-temporal` repo.